dev-libs/libflatarray/libflatarray-0.2.0.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Do you really need 0.2.0 in tree anymore? Can it just be removed in favor or 0.3.0?

Reply:
No, version 0.2.0 is needed as dependency for libgeodecomp.